css/sponsors.css

```diff
@@ -124,15 +133,18 @@ h1 {
 
 .tile > img {
     max-height: 100px;
-    max-width: 200px;
-}
-
-.tile:hover {
-    background-color: rgba(0, 0, 0, 0.1);
-    -webkit-transition: all 0.1s ease-in-out;
+    max-width: 175px;
     transition: all 0.1s ease-in-out;
 }
-
+.col-sm-2 img{
+    height: 100%;
+    width: 100%;
+    max-width: 100%;
+    max-height: 100%;
+}
+.col-sm-2{
+    padding: 5px;
+}
```

Comment:
Small change that is no longer needed; will remove it in next commit to keep it concise